apps/src/applab/dataBrowser/dataStyles.js

```diff
@@ -66,6 +66,30 @@ const input = {
   padding: '4px 6px'
 };
 
+const viewHeader = {
+  marginTop: 10,
+    marginBottom: 10,
+    marginLeft: 0,
+    marginRight: 0,
+};
+
+const backLink = {
+  fontSize: 17.5,
+};
+
+const debugLink = {
+  float: 'right',
+};
+
+const debugData = {
+  backgroundColor: color.lightest_gray,
+  borderRadius: 10,
+  border: `1px solid ${color.light_gray}`,
+  fontFamily: 'monospace',
+  padding: 10,
+  whiteSpace: 'pre-wrap',
+};
```

Comment:
Elsewhere we've tended to have a single `styles` object, with each of these items as children, i.e.
```
const styles = {
  backLink: {
    fontSize: 17.5
  },
  debugLink: {
    float: 'right'
  }
  ...
}